### ISSUE TITLE
Help: how to apply userChrome flags listed on the Tips page

### ISSUE THREAD

**Issue by virtualdj** [black7375/Firefox-UI-Fix#355]
Hi, sorry for the dumb question: I would like to hide the MenuBar icons like you suggested on issue #317.
I'm using Firefox 99 with [Lepton-Photon-Style.zip](https://github.com/black7375/Firefox-UI-Fix/releases/download/v4.6.2/Lepton-Photon-Style.zip) v4.6.2 but I don't know how to apply the two flags listed [here](https://github.com/black7375/Firefox-UI-Fix/wiki/Tips#remove-menubar-icon):

> Apply
    `userChrome.padding.global_menubar` to `false`
    `userChrome.icon.global_menubar` to `false`

I tried editing the `user.js` file and appending these lines at the end:

```javascript
// Disable menu bar icons
user_pref("userChrome.padding.global_menubar", false);
user_pref("userChrome.icon.global_menubar", false);
```

but after restarting Firefox the menu bar icons are still there. I also tried to add them as flags with `about:config` and restarting, but no luck.

If, instead, I edit `userChrome.css` directly and comment-out these two sections:
https://github.com/black7375/Firefox-UI-Fix/blob/36e9c94844fee2417662251cbd50c2b874d5b576/userChrome.css#L4565
https://github.com/black7375/Firefox-UI-Fix/blob/36e9c94844fee2417662251cbd50c2b874d5b576/userChrome.css#L4615-L4627
then after restarting the icons are gone (correctly). Unfortunately, this means that after every update I had to edit the CSS, so I'm asking which is the correct way to apply the flags listed on the Tips page. Can you please clarify?

**Comment by black7375** [comment]
Currently, you must use legacy tips.
I'm doing documentation of the version to be released later this month. #351

I'll make a clear announcement.

**Comment by virtualdj** [comment]
> Currently, you must use legacy tips. I'm doing documentation of the version to be released later this month. #351

Ah OK, make sense then!

> To apply the flag, download theme from [`dev`](https://github.com/black7375/Firefox-UI-Fix/tree/dev).

I've just tried and it works using this method, thanks. Just two "quircks":

1.  The supplied `user.js` has `userChrome.icon.global_menubar`:
https://github.com/black7375/Firefox-UI-Fix/blob/c453ef43a699759f55800a5d266a89ac11321b2b/user.js#L158
but is missing `userChrome.padding.global_menubar` (I've added below the `userChrome.padding` section)
https://github.com/black7375/Firefox-UI-Fix/blob/c453ef43a699759f55800a5d266a89ac11321b2b/user.js#L128-L139

1. I've commented the lines with `// Original` and `// Proton` to have the **Photon** theme selected but noticed the tab separators are not the same then the one using [Lepton-Photon-Style.zip](https://github.com/black7375/Firefox-UI-Fix/releases/download/v4.6.2/Lepton-Photon-Style.zip) out-of-the-box:

    ![Tab separators](https://user-images.githubusercontent.com/3251704/162573546-7c5da904-0a1f-4f88-9a76-d4b133446e0f.png)

**Comment by black7375** [comment]
Thank you for telling me.

> 1. The supplied user.js has userChrome.icon.global_menubar:

I have not yet been merged from document related branches. https://github.com/black7375/Firefox-UI-Fix/commit/0b36c6928cfa07a8cb9fdf7b90f15ec9ec6eec48
There may be some more changes to the API in the future. (unstable yet)

> I've commented the lines with // Original and // Proton to have the Photon theme selected but noticed the tab separators are not the same then the one using [Lepton-Photon-Style.zip](https://github.com/black7375/Firefox-UI-Fix/releases/download/v4.6.2/Lepton-Photon-Style.zip) out-of-the-box.

Thank you. I'll check it out.
